project/.travis/before_install_test.sh.twig

```diff
@@ -5,9 +5,16 @@ if [ "${TRAVIS_PHP_VERSION}" != "hhvm" ]; then
     mv "$HOME/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini" /tmp
     echo "memory_limit=3072M" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
 
+    {% if 'mongodb' in services %}
     if [ ${TRAVIS_PHP_VERSION} '<' '7.0' ]; then
         echo "extension=mongo.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+    else
+        echo "extension=mongodb.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+
+        # Backwards compatibility with old mongo extension
+        composer require "alcaeus/mongo-php-adapter" --no-update
```

Comment:
It provides compatiblity with old PHP, because the extension name has changed. You can't add directly on composer.json require or require-dev because it depends on php version. Usually it is on suggest section, but that is not enough